### ISSUE TITLE
1.0.0 error in css and js links in template

### ISSUE THREAD

**Issue by paul-frost** [engram-design/ImageResizer#25]
The includes in templates/settings/index.html are missing the /resources/ part of the address.
But having corrected those the tab bar's not quite right, with an additional Settings tab that I don't think should be there.

**Comment by engram-design** [comment]
Have you cleared your caches? You don't have to manually include `/resources/` when using `includeCssResource` or `includeJsResource`.

There should be two tabs - Logs and Settings. Are you able to post a screenshot of your browser console? Please note the CSS/JS files are capitalized as well.

**Comment by paul-frost** [comment]
I thought I had cleared the cache, but the page looked completely unstyled, which is what made me think the css wasn't getting loaded. Then after I modified the addresses it looked almost normal but gave me the attached screenshot. I've now put the links back to the original code and it looks ok, so it must have been a cache issue. Apologies for making a false claim.
![screen shot imagsizer](https://cloud.githubusercontent.com/assets/384378/24334986/bb6ce6a2-126c-11e7-9e7d-93d71ee186e8.jpg)
